### ISSUE TITLE
Allow printing Nest errors when something goes wrong

### ISSUE THREAD

**Issue by gmenih341** [jmcdo29/nestjs-commander#6]
So I guess I'm breaking the ice and creating the first issue 😅 

I really like this package, it's something I've been looking for in the past, and it works exactly as I had hoped! There's just a small issue with error logging though. Since `logger` is hardcoded to `false` when creating the Nest application, it means that any error messages are also ignored.. Nest has really nice error logs, so it would be nice to keep them. Especially, since it seems to be easy (at least for me) to mess up the providers somehow, and not know what went wrong.

Would you agree to support `NestApplicationContextOptions`, or at least the `logger` option as a second parameter to the `CommandFactory.run`? I would be happy to create a PR to add this :)

**Comment by jmcdo29** [comment]
>So I guess I'm breaking the ice and creating the first issue 😅

Yes! Do it!

---

We can make the logger optional, but default to false. I agree it would be great to have the ability to log the errors if something goes wrong, but definitely don't want them by default. If we have them on by default then when the command successfully runs we'd get logs like

```
[Nest] 41846   - 01/19/2021, 11:04:36 AM   [NestFactory] Starting Nest application...
[Nest] 41846   - 01/19/2021, 11:04:36 AM   [InstanceLoader] FooModule dependencies initialized +29ms
[Nest] 41846   - 01/19/2021, 11:04:36 AM   [InstanceLoader] AppModule dependencies initialized +0ms
[Nest] 41846   - 01/19/2021, 11:04:36 AM   [InstanceLoader] BarModule dependencies initialized +0ms
```
every time which isn't a good end user experience.

Always happy to accept PRs though!

**Comment by gmenih341** [comment]
Cool! It was super easy to add it, here's the PR: https://github.com/jmcdo29/nestjs-commander/pull/7

**Comment by jmcdo29** [comment]
Closed by #7
